sdk/tables/azure-data-tables/swagger/README.md

```diff
@@ -33,7 +31,80 @@ add-context-parameter: true
 models-subpackage: implementation.models
 context-client-method-parameter: true
 service-interface-as-public: true
-enable-xml: true
```

Comment:
Why was this flag removed?

Reply:
Doesn't exist anymore. XML serialization logic is now based on the model being marked as XML.